src/services/expensesService.js

```diff
@@ -0,0 +1,79 @@
+let expenses = [];
+
+const init = () => {
+  expenses = [];
+};
+
+const getExpenses = (userId, categories, from, to) => {
+  let filteredExpenses = [...expenses];
+
+  if (userId) {
+    filteredExpenses = filteredExpenses.filter(
+      (item) => item.userId === Number(userId),
+    );
+  }
+
+  if (categories) {
+    filteredExpenses = filteredExpenses.filter(
+      (item) => item.category === categories,
+    );
+  }
+
+  if (from || to) {
+    filteredExpenses = filteredExpenses.filter(
+      (item) => item.spentAt >= from && item.spentAt <= to,
+    );
+  }
```

Comment:
DRY. Use `filteredExpenses = filteredExpenses.filter` once and move all `if`s inside